patches/server/1045-Add-missing-fishing-event-state.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SoSeDiK <mrsosedik@gmail.com>
+Date: Wed, 1 May 2024 07:44:50 +0300
+Subject: [PATCH] Add missing fishing event state
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/projectile/FishingHook.java b/src/main/java/net/minecraft/world/entity/projectile/FishingHook.java
+index 35299affb699d745804a3b60cb78b9323f3e068e..bca9b5e9c7e57a81f4b9691a22e44d485cd223c1 100644
+--- a/src/main/java/net/minecraft/world/entity/projectile/FishingHook.java
++++ b/src/main/java/net/minecraft/world/entity/projectile/FishingHook.java
+@@ -410,6 +410,15 @@ public class FishingHook extends Projectile {
+                     this.fishAngle = Mth.nextFloat(this.random, this.minLureAngle, this.maxLureAngle);
+                     this.timeUntilHooked = Mth.nextInt(this.random, this.minLureTime, this.maxLureTime);
+                     // CraftBukkit end
++                    // Paper start - Add missing fishing event state
++                    if (this.getPlayerOwner() != null) {
++                        var playerFishEvent = new PlayerFishEvent((Player) this.getPlayerOwner().getBukkitEntity(), null, (FishHook) this.getBukkitEntity(), PlayerFishEvent.State.LURED);
```

Comment:
no var please, use a qualified import

Reply:
Is there a specific policy regarding `var`s? I don't see anything in contributing.md and Paper uses it in a bunch of other places similar to this one.
It's a constructor, so it should be obvious what's returned?